ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -84,38 +93,60 @@ async def test_get_data(ofrak_server, aiohttp_client, hello_world_elf):
     create_resp = await client.post(
         "/create_root_resource", params={"name": "test"}, data=hello_world_elf
     )
-    body = await create_resp.json()
-    resp = await client.get(f"/{body['id']}/get_data")
+    create_body = await create_resp.json()
+    resp = await client.get(f"/{create_body['id']}/get_data")
     assert resp.status == 200
+    resp_body = await resp.read()
+    assert resp_body is not None
```

Comment:
In situations like these, it's more useful to assert that `resp_body == hello_world_elf`. We know what the data is supposed to be, so we should confirm that it matches!

This comment applies to most of the improved tests – if we know what to expect, we should confirm in the tests.